src/src_V2/components/tooltip/TooltipText.jsx

```diff
@@ -0,0 +1,19 @@
+import { useRef } from 'react';
+import { Tooltip } from '@mui/material';
+
+import { useHoverTooltip } from '../../hooks';
+import styles from './TooltipText.module.css';
+
+export default function TooltipText({ text }) {
+  const textElementRef = useRef();
+
+  const hoverStatus = useHoverTooltip(text, textElementRef);
+
+  return (
+    <Tooltip title={text} interactive="true" disableHoverListener={!hoverStatus}>
+      <div ref={textElementRef} className={styles.tooltip}>
+        {text}
+      </div>
+    </Tooltip>
+  );
+}
```

Comment:
note just for have a discussion for this component with you

Reply:
Is this also global one?

Reply:
Yes, it is also a global component. It is inside the correct directory right?

Reply:
yeah it is, It seems good one but i just whant to know why it is a global?

Reply:
It can be reused. This component works in cooperation with useHoverTooltip custom hook. So if the element doesn't fit its content, three dots and a tooltip will be shown. Feel free to customize it according to your use case, maybe it will be better to send the width of the element from the outside.

Reply:
Got it, ok great.